docs/Advanced Configuration/External Database.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_position: 8
+id: External Database
+title: External Database
+---
+
+# Using an External Database
+
+It is possible to use your own external database with Stirling PDF rather than the default H2 database if you wish. 
+PostgreSQL is currently supported with others soon to follow.
+
+### Setting Up External Database Configuration
+You can configure the new `Datasource` property in your `settings.yml` to connect to your external database:
+
+```yaml
+  datasource:
+    enableCustomDatabase: false
+    customDatabaseUrl: jdbc:postgresql://localhost:5432/postgres
+    username: postgres
+    password: postgres
+    type: postgresql
+    hostName: localhost
+    port: 5432 
+    name: postgres
+```
+
+- `enableCustomDatabase`: Set this property to `true` to enable use of the custom database
+- `customDatabaseUrl`: Enter the database connection url for the database here. **Note: If you set the `customDatabaseUrl` you do not need to set the type, hostName, port and name, they will all be automatically derived from the url.**
+- `username`: The username for the database
+- `password`: The password for the database
+
+If you would like more fine-grained control of the database connection, you can also use the following properties:  
+
+#### Fine-grained Database Configuration
+- `type`: The database type. Available options are `h2` and `postgresql`
+- `hostName`: The host name of the database connection url (e.g. 'localhost')
+- `port`: The port number of the database connection url (e.g. 8080)
+- `name`: The name of the custom database. This should match the name you have set for your database
+
+### Docker Based Configuration
+You can also configure an external database within using Docker. 
```

Comment:
I think rather tahn doing a THis or That approach with settings.yml and docker it might be better to use a tabbed function
here is a example
https://github.com/Stirling-Tools/Stirling-Tools.github.io/commit/593e33dad882f387b50b58b59158774cae081d82

https://docs.stirlingpdf.com/Advanced%20Configuration/UI%20Customisation